client/src/components/Member.js

```diff
@@ -5,30 +5,114 @@ import CardContent from '@mui/material/CardContent';
 import Typography from '@mui/material/Typography';
 import Button from '@mui/material/Button';
 import Stack from '@mui/material/Stack';
+import { CardMedia, Grid } from '@mui/material';
+import { useState } from 'react';
+import ManageMemberModal from '../modals/ManageMemberModal';
 
-function Member({member}) {
+function Member(props) {
+  const [openManageMemberModal, setOpenManageMemberModal] = useState(false);
+
+  // Delete the memner from the association
+  const deleteMember = async () =>
+  {
+    fetch('/api/delete/member/' + props.member.id, {
```

Comment:
the param should be props.currentUser.id I guess, noticed this when trying to delete a member. Other than this, everything looks really good! OK to merge after fixing this. 🤝